### PR TITLE
Add a thunderbird app for TB copy

### PR DESCRIPTION
**Pull request by Pomax**
This is an alternate take on https://github.com/mozilla/donate-wagtail/pull/499, introducing thunderbird-specific templates that are loaded "instead of" the regular templates by having a separate thunderbird app that gets loaded into INSTALLED_APPS as well as the TEMPLATES dirs, contingent on the same `THUNDERBIRD_INSTANCE` variable that @TheoChevalier went with.

The benefit of this approach is that the only files that need updating are template files, without any model or view changes.

In order to keep things easy to maintain going forward, some of the templates have been split up into a `..._master.html` and a corresponding one-liner `.html` file that extends the master file and does literally nothing else. However, the thunderbird app can then "hijack" those files by declaring templates with the same name and path, so they get used instead, while still being able to extend the master file with new block content. Without that split, the thunderbird templates would have been full .html copies with content replacement, which would be much harder to maintain.

@alanmoo @TheoChevalier how does this one feel?

### CODE REVIEW

File: donate/settings.py [hunk 1]
```diff
@@ -61,6 +61,8 @@
     RECAPTCHA_ENABLED=(bool, False),
     SENTRY_DSN=(str, None),
     HEROKU_RELEASE_VERSION=(str, None),
+    # Instance vars
+    THUNDERBIRD_INSTANCE=(bool, True),
```
Comment (cadecairos) [inline donate/settings.py]: Ideally we can use a specific configuration for doing this once #500 lands

File: donate/templates/fragments/donate_form_disclaimer.html [hunk 2]
```diff
@@ -1,29 +1,3 @@
...
-    <p>
-        {% blocktrans with faq_url='/faq' email='donate@mozilla.org' trimmed %}
-            Problems donating? Visit our <a href="{{ faq_url }}">FAQ</a> for answers to most common questions. Still have problems? <a href="mailto:{{ email }}">Send us an email</a>.
-        {% endblocktrans %}
-    </p>
-    <p>
-        {% blocktrans %}Contributions go to the Mozilla Foundation, a 501(c)(3) organization based in Mountain View, California, to be used in its discretion for its charitable purposes. They are tax-deductible in the U.S. to the fullest extent permitted by law.{% endblocktrans %}
-    </p>
-</footer>
+{# There are no overrides in this template, but alternative apps can template-overide this file (e.g. thunderbird) #}
```
[the remark this answers is not in the document]
Comment (Pomax) [reply donate/templates/fragments/donate_form_disclaimer.html]: Mostly for ease of maintenance: keeping the main file one thing means dev and design don't need to mentally piece together multiple files to make changes to the master file.

[the remark this answers is not in the document]
Comment (cadecairos) [reply donate/templates/fragments/donate_form_disclaimer.html]: Had a chat with Pomax to clarify the reasoning for keeping so much content in the master template, and it makes sense to me as we move forward.

**Comment by TheoChevalier** [comment]
String changes LGTM!

**Pull request merged**